### ISSUE TITLE
Qt framework files in the root of the dmg

### ISSUE THREAD

**Issue by The-Compiler** [qutebrowser/qutebrowser#3080]
The macOS .dmg has things like `QtCore.framework` at the root of the `.app` (where `Contents` is). They probably don't belong there.

**Comment by dvs1** [comment]
Are you using [macdeployqt](http://doc.qt.io/qt-5/osx-deployment.html)?
The correct location is`Contents/Frameworks`.

**Comment by The-Compiler** [comment]
Nope, [PyInstaller](http://www.pyinstaller.org/). I think they are in the correct location as well.

**Comment by The-Compiler** [comment]
Seems to be intentional: https://github.com/pyinstaller/pyinstaller/blob/4b9fd01d7dc5a0aae3ba7c100437547915464f0e/PyInstaller/utils/hooks/qt.py#L727-L751
